spark/src/main/java/org/apache/zeppelin/spark/SparkInterpreter.java

```diff
@@ -563,6 +603,17 @@ public void open() {
         intp.interpret("import sqlContext.sql");
         intp.interpret("import org.apache.spark.sql.functions._");
       }
+
+      if (useMahout()) {
+        intp.interpret("import org.apache.mahout.math._");
+        intp.interpret("import org.apache.mahout.math.scalabindings._");
+        intp.interpret("import org.apache.mahout.math.drm._");
+        intp.interpret("import org.apache.mahout.math.scalabindings.RLikeOps._");
+        intp.interpret("import org.apache.mahout.math.drm.RLikeDrmOps._");
+        intp.interpret("import org.apache.mahout.sparkbindings._");
+        intp.interpret("implicit val sdc: "
+                + "org.apache.mahout.sparkbindings.SparkDistributedContext = sc2sdc(sc)");
```

Comment:
sc2sdc is implicit, if the entire sparkbindings._ is imported, then this should be just 

    <...> = sc